src/frontend/src/env/tokens.btc.env.ts

```diff
@@ -15,7 +15,7 @@ export const BTC_MAINNET_TOKEN: Token = {
 	standard: 'bitcoin',
 	category: 'default',
 	name: 'Bitcoin',
-	symbol: 'BTC',
+	symbol: BTC_MAINNET_SYMBOL,
```

Comment:
While creating a new token, I noticed that `BTC_MAINNET_TOKEN` doesn't use `BTC_MAINNET_SYMBOL` as a value for `symbol`. Same with `BTC_TESTNET_TOKEN` below. I update it, but please let me know if there was a specific reason for that.

Reply:
Good catch! I saw you corrected both in `mainnet` and `testnet`, thank you!

Reply:
Strange that it is done in the same way for `ETH` and `SepoliaETH`... maybe @peterpeterparker has a better insight?